common/src/main/java/com/wynntils/features/user/ItemGuessFeature.java

```diff
@@ -6,14 +6,14 @@
 
 import com.google.common.collect.ImmutableList;
 import com.wynntils.core.config.properties.ConfigOption;
-import com.wynntils.core.config.properties.Configurable;
+import com.wynntils.core.config.properties.ConfigurableInfo;
 import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.features.properties.FeatureInfo;
 import com.wynntils.core.features.properties.FeatureInfo.Stability;
 import com.wynntils.core.webapi.WebManager;
 
 @FeatureInfo(stability = Stability.STABLE)
-@Configurable(category = "Item Tooltips")
+@ConfigurableInfo(category = "Item Tooltips")
```

Comment:
What do you think about making categories string constant in some files? What if we would want to change `Item Tooltips` to `whatever new name` and you would need to do that manually (or using IDE's find and replace). I think something like `Categories.ITEM_TOOLTIPS` would fit here better.

Reply:
Actually, this won't be a problem if we use translations for config stuff.

Reply:
Further tracked in #94.

Reply:
Even if we use translations, I think it makes sense to have this an enum or a const defined in a class. It just makes it easier to make sure a typo don't ruin things, or do quickly go to all members of the category from the IDE. In general, using "raw" strings like this is not a good idea.